wp-content/themes/engage/templates/page-tool.twig

```diff
@@ -5,33 +5,52 @@
 
 		<div class="article__content">
 			<article class="quiz-creator-product-page" id="tool-landing-page">
-				<section class="hero aqua" style="margin-top: -36px;">
+				<section class="hero aqua" style="margin-top: -36px; margin-bottom:0px;">
```

Comment:
Unless we have to, it's ideal to reuse existing classes and leave CSS styles in our SCSS files. Can you remove the `style=""` definitions from this file and use existing CSS classes where possible. If there isn't a CSS class that already is close to what you're after, you can add a new `.class` definition to the appropriate file.